src/pages/Search/ResultFilters/FilterCategory.js

```diff
@@ -18,12 +19,20 @@ export function FilterCategory({
   filterValues,
   activeValues,
   formatValue = x => x,
-
   onToggleFilter,
 }) {
   const [query, setQuery] = React.useState('');
+  const [savedQuery, setSavedQuery] = useLocalStorage(
```

Comment:
Can we save this in a different place? I think we talked about refs or the store?

Reply:
I think we might have to use this temporarily since the filters are being regenerated dynamically. We would have to refactor the filter to persist the instance of the FilterCategory in order to keep it in a ref or state.

Reply:
We would have to prevent refreshing the page

Reply:
hmm are we refreshing the whole page when filters are applied?

Reply:
yeah whenever the querystring changes we reload the page